Osiris/Hacks/Misc.cpp

```diff
@@ -791,3 +791,23 @@ void Misc::purchaseList(GameEvent* event) noexcept
         ImGui::End();
     }
 }
+
+void Misc::doorSpam(UserCmd* cmd) noexcept {
+
+    if (!localPlayer || !config->misc.doorSpam || localPlayer->isDefusing())
+        return;
+
+    static bool doorSpam = true;
+    constexpr float doorRange{ 200.0f };
+    Vector viewAngles{ cos(degreesToRadians(cmd->viewangles.x)) * cos(degreesToRadians(cmd->viewangles.y)) * doorRange,
```

Comment:
Use Vector::fromAngles()

Reply:
How? Because IDK, sry

Reply:
Can you show me example?

Reply:
https://github.com/danielkrupinski/Osiris/blob/92ec15dc50a745e043c1e2d3a77faf9e83e141f4/Osiris/Hacks/Misc.cpp#L302